students/john_navitsky/session02/series.py

```diff
@@ -0,0 +1,49 @@
+
+""" Functions that return Nth entry of a sum sequence. """
+
+def sum_series(nth=1,sequence=[0,1]):
```

Comment:
This is the non-recursive way to do it -- which, in fact, is probably better performing (though if you don't need the whole sequence, it may store extraneous stuff.

But do make sure you know how to do it recursively as well -- sometimes that makes the logic much simpler.